docs/source/top_level_elements/location.rst

```diff
@@ -0,0 +1,203 @@
+
+Distinct Cases
+--------------
+
+physicalLocation with explicit uri
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/egypt:79/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2017033007">Frank H. McClung Museum of Natural History and Culture</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2017033007> .
+
+physicalLocation as string (special collections)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/roth:4245/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennessee, Knoxville. Special Collections</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2014027633> .
+
+physicalLocation as string (libraries)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**The University of Tennessee Libraries, Knoxville** 574 instances
+
+https://digital.lib.utk.edu/collections/islandora/object/fbpro:94819/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>The University of Tennessee Libraries, Knoxville</physicalLocation>
+    </location>
+
+**University of Tennessee Knoxville. Libraries** 664 instances
+
+https://digital.lib.utk.edu/collections/islandora/object/tatum:609/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennessee Knoxville. Libraries</physicalLocation>
+    </location>
+
+**University of Tennesse Knoxville. Libraries** 397 instances
+
+https://digital.lib.utk.edu/collections/islandora/object/tdh:8781/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennesse Knoxville. Libraries</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/n80003889> .
+
+physicalLocation and shelfLocator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/scopes:1258/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2014027633">University of Tennessee, Knoxville. Special Collections</physicalLocation>
+        <shelfLocator>Box 5, Folder 8</shelfLocator>
+    </location>
+
+In most cases, especially those under our purview, we will likely opt to drop box and folder number references.
```

Comment:
Can we specify the exact collections this does not apply to so that we can drop accordingly? I know you have included some arrowmont records below that seem to show when we are keeping shelfLocator, but it would be good to be very specific. I can help come up with a list if you'd like.

Reply:
Perhaps this is counter to the point of analysis, but I'm not sure if I can really breakdown exactly where that threshold exists. This isn't really an XPath decision as much as one that would need to be curated by someone with care for a records, the collections, and the physical repository they belong to. Because of that, I'm thinking of this analysis doc as a stepping stone before the mapping decisions if the cases (to drop shelfLocator literals or translate to RDF using opaquenamespace predicates) do come about, not what records will go into which pile.

At the same time, maybe I have missed something in my analysis that would be helpful to mapping.

I'll review my [spreadsheet](https://docs.google.com/spreadsheets/d/1ghf0sP2zX2mC9aEgs7JmvhY9L5fq5YQcNWQf7WJV0Ts/edit?usp=sharing) more thoroughly to see if there are cases I might be missing.